dashboard-ui/src/components/TextBasedScenarios/alignmentID.json

```diff
@@ -0,0 +1,60 @@
+{
+    "adeptAlignmentIDs": [
+        "ADEPT-DryRun-Moral judgement-0.1",
+        "ADEPT-DryRun-Ingroup Bias-0.1",
+        "ADEPT-DryRun-Ingroup Bias-0.0",
+        "ADEPT-DryRun-Moral judgement-0.0",
+        "ADEPT-DryRun-Moral judgement-0.7",
+        "ADEPT-DryRun-Ingroup Bias-0.7",
+        "ADEPT-DryRun-Ingroup Bias-0.6",
+        "ADEPT-DryRun-Moral judgement-0.6",
+        "ADEPT-DryRun-Ingroup Bias-0.9",
+        "ADEPT-DryRun-Moral judgement-0.9",
+        "ADEPT-DryRun-Ingroup Bias-0.5",
+        "ADEPT-DryRun-Moral judgement-0.5",
+        "ADEPT-DryRun-Moral judgement-0.4",
+        "ADEPT-DryRun-Moral judgement-1.0",
+        "ADEPT-DryRun-Ingroup Bias-1.0",
+        "ADEPT-DryRun-Ingroup Bias-0.4",
+        "ADEPT-DryRun-Moral judgement-0.8",
+        "ADEPT-DryRun-Ingroup Bias-0.8",
+        "ADEPT-DryRun-Ingroup Bias-0.3",
+        "ADEPT-DryRun-Moral judgement-0.3",
+        "ADEPT-DryRun-Moral judgement-0.2",
+        "ADEPT-DryRun-Ingroup Bias-0.2"
```

Comment:
Just a minor nit, but sorting this list would make it less error-prone (easier to see errors), although everything is indeed there.